src/__experimental__/components/select/select.spec.js

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import Select from './select.component';
+import guid from '../../../utils/helpers/guid';
+import Events from '../../../utils/helpers/events';
+
+jest.mock('../../../utils/helpers/guid');
+guid.mockImplementation(() => 'guid-12345');
+
+describe('Select', () => {
+  const renderWrapper = ({ props, type = mount } = {}) => (
+    type(
+      <Select { ...props }>
+        <div>1</div>
+        <div>2</div>
+        <div>3</div>
+      </Select>
+    )
+  );
+
+  // an example of a single value
+  const singleValue = {
+    value: '1',
+    text: 'Orange'
+  };
+
+  // an example of a multi value
+  const multiValue = [{
+    value: '1',
+    text: 'Orange'
+  }, {
+    value: '2',
+    text: 'Blue'
+  }, {
+    value: '3',
+    text: 'Red'
+  }];
+
+  // utility functions to fetch various elements from the wrapper
+  const fetchList = wrapper => wrapper.find('SelectList');
```

Comment:
Consider using the naming convention `[propName]Of = componentName => {}` so that in the tests you get that nice `expect(pillsOf(component)).toBe(whatever)`.

Reply:
done 👍